### ISSUE TITLE
Implement the ability for commands to be admin-ediable in their entirety

### ISSUE THREAD

**Issue by MegaMasterX** [MegaMasterX/ColonelBot-v4#31]
Commands, such as !guides, should be admin editable and operate off of a serializable object that's read/written to disk to eliminate the need to do bot code changes in order to update the text of command outputs.

**Comment by MegaMasterX** [comment]
Implemented in [894d2e0](https://github.com/MegaMasterX/ColonelBot-v4/commit/894d2e0e70c7c08e993f3b87f509cf5b3f753cc3). This will need to be tested on the N1GP unless a role update is done.  

The Moderator Only attribute needs to be tested as a prerequisite to this being integrated and considered done.  Currently there are 2 roles called "Moderator" on the N1GP Discord and I've requested the color changing role be renamed so I can remove the hardcoded ulong role IDs. https://discord.com/channels/132102389455912960/256865775070085131/1108719101569925230
